package.json

```diff
@@ -31,6 +32,7 @@
     "body-parser": "1.18.3",
     "cors": "2.8.4",
     "dotenv": "6.0.0",
+    "download": "^7.1.0",
```

Comment:
What did you use this for?

Reply:
Also the version is not locked

Reply:
Oh! really sorry. it was an oversight because the package was not needed. It was just an update to my NPM Version. I will amend that immediately.